.murdock.yml

```diff
@@ -12,7 +12,7 @@ pr:
     readable_name: Documentation preview
 
 commit:
-  skip_keywords: ["ci_skip", "murdock_skip"]
+  skip_keywords: ["ci_skip", "murdock_skip", "[ci skip]" ]
```

Comment:
@aabadie murdock is building this despite `[ci skip]` being in the commit subject:

![image](https://user-images.githubusercontent.com/4679640/204241471-668ba84d-af08-49f9-8a44-ecbbc05cad33.png)

Reply:
can you try "ci_skip" (which is already defined) ?

Reply:
I could on a random PR, that specific commit was generated by bors (which I don't control).

Reply:
I've added "ci_skip" to the topmost commit here: https://github.com/RIOT-OS/RIOT/pull/18586
It got correctly skipped!